p4-16/psa/PSA.mdk

```diff
@@ -451,25 +451,23 @@ sequence numbers in packets, for example, use Registers instead
 
 Direct counters are counters associated with a particular P4 table,
 and are implemented by the extern `DirectCounter`.  There are also
-indirect counters, which are implemented by the extern `Counter` (the
-name 'indirect' does not imply that there is any indirection involved
-in the implementation, only that they are not direct counters).  The
-primary differences between direct counters and indirect counters are:
+indexed counters, which are implemented by the extern `Counter`.  The
+primary differences between direct counters and indexed counters are:
```

Comment:
I don't mind renaming Counter -> IndexedCounter and Meter -> IndexedMeter if there are no objections.